R/bs-theme-preview.R

```diff
@@ -268,18 +256,21 @@ run_with_themer <- function(appDir = getwd(), ...) {
 #' @export
 bs_themer <- function() {
   session <- shiny::getDefaultReactiveDomain()
-
-  if (!is.null(bs_theme_get()) && "3" %in% theme_version(bs_theme_get())) {
-    stop("Interactive theming for Bootstrap 3 Sass isn't yet supported")
-  }
-
   if (is.null(session)) {
     stop(call. = FALSE, "bootstraplib::bs_themer() must be called from within a ",
-      "Shiny server function")
+         "Shiny server function")
   }
   if (!identical("ok", session$ns("ok"))) {
     stop(call. = FALSE, "bootstraplib::bs_themer() must be called from within a ",
-      "top-level Shiny server function, not a Shiny module server function")
+         "top-level Shiny server function, not a Shiny module server function")
+  }
+  theme <- shiny::getShinyOption("bs_theme")
```

Comment:
@wch can/should we do something else that's better?

Reply:
Now that I look at this in more detail, I think it's OK, since this is so tied with Shiny's internals. But didn't the option get renamed to something else?

Reply:
Great, yea, I handled the rename in https://github.com/rstudio/bootstraplib/pull/114/commits/c9293e9b9f461933b4301ccbd395f16e7473aa5a